### DISCUSSION TITLE
Set default button shape ?

### DISCUSSION THREAD

**Discussion by LoisDuplain** [ant-design/ant-design#50201] [Q&A]
Hello,

Is there a way to set the default shape of every Antd buttons in my app ?

Thanks

**Comment by crazyair** [comment]
HOC?

**Reply by LoisDuplain** [reply]
What does this mean ?

**Comment by relativepathson** [comment]
@LoisDuplain wrap your entry component with the ConfigProvider. See the attached example.
<img width="960" alt="antd-example" src="https://github.com/user-attachments/assets/8c817faf-d477-46ea-9e28-1d5fab5cc44f">
